includes/class-wc-payments-token-service.php

```diff
@@ -120,13 +123,14 @@ public function woocommerce_get_customer_payment_tokens( $tokens, $user_id, $gat
 			$stored_tokens = [];
 
 			foreach ( $tokens as $token ) {
-				if ( WC_Payment_Gateway_WCPay::GATEWAY_ID === $token->get_gateway_id() ) {
+				if ( in_array( $token->get_gateway_id(), self::REUSABLE_GATEWAYS, true ) ) {
 					$stored_tokens[ $token->get_token() ] = $token;
 				}
 			}
 
-			$payment_methods = [ [] ];
-			foreach ( WC_Payments::get_gateway()->get_upe_enabled_payment_method_ids() as $type ) {
+			$retrievable_payment_method_types = [ Payment_Method::CARD, Payment_Method::SEPA ];
+			$payment_methods                  = [];
+			foreach ( $retrievable_payment_method_types as $type ) {
```

Comment:
```suggestion
			$retrievable_payment_method_types = [ Payment_Method::CARD ];
			if ( in_array( Payment_Method::SEPA, WC_Payments::get_gateway()->get_upe_enabled_payment_method_ids(), true ) {
				$retrievable_payment_method_types[] = Payment_Method::SEPA;
			}
			$payment_methods                  = [];
			foreach ( $retrievable_payment_method_types as $type ) {
```
Just a curt caveat that we only to request the SEPA payment methods when SEPA is actually enabled at checkout. Otherwise even when SEPA is disabled, I believe we will still show users their saved SEPA tokens on the Accounts page.

Reply:
Thanks for catching that. Makes total sense. -> https://github.com/Automattic/woocommerce-payments/pull/4670/commits/d3e9a3dc08c21b61185b0e8fff20f513629830f5